changelog/16699.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
Is this considered a bugfix? Or just an improvement?

Reply:
Good catch, it's more of an improvement now.